src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -572,6 +572,25 @@ public function text()
     }
 
     /**
+     * Returns the flattened node value of the first node of the list.
+     *
+     * Surrounding spaces/newlines are removed,
+     * and consecutive spaces/newlines are reduced to one space.
+     *
+     * @return string The flattened node value
+     *
+     * @throws \InvalidArgumentException When current node is empty
+     */
+    public function flatText()
```

Comment:
You're flattening `text()`.. it's an action :) i'd also favor `flattenText()` here.